notebooks/4_SparseKernelMethods.ipynb

```diff
@@ -359,15 +361,15 @@
    "source": [
     "fig, axes = plt.subplots(1,2, figsize=dbl_fig)\n",
     "\n",
-    "ref_kpca = KPCA(n_PC=n_PC, kernel_type=kernel_type)\n",
-    "ref_kpca.fit(X_train, K=K_train)\n",
-    "xref = ref_kpca.transform(X_test, K=K_test)\n",
+    "ref_kpca = KernelPCA(n_components=n_PC, kernel='rbf',gamma = 1.0, fit_inverse_transform=True)\n",
```

Comment:
As I said,  using `kernel= "precomputed"` will cause inconvenience. Here we have to use `inverse_transform` method, but it is not compatible with `kernel= "precomputed"`. I have to set `kernel='rbf',gamma = 1.0` to use `inverse_transform` method.
![image](https://user-images.githubusercontent.com/56018055/99859671-04759880-2b91-11eb-95fe-cca2d2e33d9a.png)
![image](https://user-images.githubusercontent.com/56018055/99859676-09d2e300-2b91-11eb-984b-92f7ae4fe054.png)

Reply:
Why not still put these in `kernel_params`?